my_game.py

```diff
@@ -99,19 +99,28 @@ class Player(arcade.Sprite):
     The player
     """
 
-    def __init__(self, center_x, center_y, lives, scale):
+    def __init__(self,
+                 center_x,
+                 center_y,
+                 lives,
+                 scale,
+                 start_speed_min,
+                 start_speed_max,
+                 start_angle_min,
+                 start_angle_max):
         """
         Setup new Player object
         """
 
         # Graphics to use for Player
         super().__init__("images/playerShip1_red.png", flipped_horizontally=True ,flipped_diagonally=True)
         self.invincibility_timer = 0
-        self.angle = 0
         self.lives = lives
         self.scale = scale
         self.center_x = center_x
         self.center_y = center_y
+        self.angle = random.randint(start_angle_min, start_angle_max)
+        self.forward(random.randrange(start_speed_min, start_speed_max))
```

Comment:
Lets use _random.uniform(start_speed_min, start_speed_max)_. That will give us a _float_ instead of an _int_.